### PR TITLE
add(questions to issue template)

### PR DESCRIPTION
**Pull request by ChrOertlin**
### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ As a [type of user],
...
+<!--
+Filled in by the Product Owner or the Stakeholders.
+-->
+
+Answer the following questions:
+- Is there currently a workaround for this issue? If so, what is it?
+- How much time would be saved by implementing this feature on a weekly basis?
+- How many users are affected by this issue?
+
+
```
Comment (ahdamin) [inline .github/ISSUE_TEMPLATE/user-story.md:28]: I noticed that the impact on the customer may affect the priority. Is that something you want to consider here?

Comment (ChrOertlin) [reply .github/ISSUE_TEMPLATE/user-story.md:28]: Yes, good idea

**Review by islean — APPROVED** [review]
I like it!

**Review by Vince-janv — APPROVED** [review]
Nice 👍

**Review by Vince-janv — COMMENTED** [review]
I just realised that the changes should be made here: https://github.com/Clinical-Genomics/.github/tree/master/.github/ISSUE_TEMPLATE if we want them to apply to the whole org

**Review by diitaz93 — APPROVED** [review]
👍

**Pull request merged**